swf/src/shapes.rs

```diff
@@ -94,12 +94,12 @@ impl Shape {
 
     #[inline]
     fn first(&self) -> LineSegment {
-        self.outline.first().unwrap()
+        *self.outline.first().unwrap()
```

Comment:
Will this still compile?

Reply:
It compiles for me (`cargo build -p pathfinder_swf`)

Reply:
without:
```
error[E0308]: mismatched types
   --> swf/src/shapes.rs:102:9
    |
101 |     fn last(&self) -> LineSegment {
    |                       ----------- expected `shapes::LineSegment` because of return type
102 |         self.outline.last().unwrap()
    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |         |
    |         expected struct `shapes::LineSegment`, found reference
    |         help: consider dereferencing the borrow: `*self.outline.last().unwrap()`
    |
    = note: expected type `shapes::LineSegment`
               found type `&shapes::LineSegment`
```